src/tools/schema2cpp.ts

```diff
@@ -57,6 +57,24 @@ export class Schema2Cpp extends Schema2Base {
     return baseName.toLowerCase().replace(/\.arcs$/, '').replace(/[._]/g, '-') + '.h';
   }
 
+  generateEntityClassName(node: SchemaNode, i: number) {
+    if (i === -1) {
```

Comment:
Probably worth documenting somewhere what the -1 value means (and maybe make it a default value for the i argument) (and maybe use null instead of -1)

Reply:
Went to null as a default value.